bip-0003.md

```diff
@@ -391,67 +389,69 @@ innovate on a level playing field. Only freely licensed contributions are accept
+under CC0-1.0 or FSFAP in addition to any other licenses to allow anyone to copy test vectors into their
+implementations without introducing license hindrances. Licenses listed in the License-Code header apply to all source directories,
+source code files, and test vectors provided with the BIP except those where a LICENSE file in a directory
 or the file header states otherwise.
 
 For example, a preamble specifying the optional License-Code header might look like:
 
     License:         CC0-1.0
     License-Code:    MIT
 
-In this case, the code in the BIP is not available under CC0-1.0, but is only available under the terms of the MIT
+In this case, the source code in the BIP is not available under Creative Commons CC0 1.0 Universal, but is only available under the MIT
 License.
 
-BIPs are not required to be *exclusively* licensed under approved terms, and may also be licensed under unacceptable
-licenses *in addition to* at least one acceptable license. In this case, only the acceptable license(s) should be listed
-in the License and License-Code headers.
-
-It is recommended that BIPs that include literal code be licensed under the same license terms as the project it
-modifies. For example, literal code intended for Bitcoin Core would ideally be licensed (or dual-licensed) under the MIT
-license terms.
+It is recommended that source code included in a BIP (whether within the text or in auxiliary files) be licensed under the same license terms as the project it
+is proposed to modify, if any. For example, changes intended for Bitcoin Core would ideally be licensed (also) under the MIT
+License.
 
 In all cases, details of the licensing terms must be provided in the Copyright section of the BIP.
 
 #### Acceptable Licenses[^licenses]
 
-* BSD-2-Clause: [OSI-approved BSD 2-clause license](https://opensource.org/licenses/BSD-2-Clause)
-* BSD-3-Clause: [OSI-approved BSD 3-clause license](https://opensource.org/licenses/BSD-3-Clause)
+Each new BIP must be made available under at least one acceptable license as listed below. BIPs are not required to be
+*exclusively* licensed under approved terms, and may also be licensed under other licenses *in addition to* at least one
+acceptable license.
+
+In other words, a new BIP must specify an SPDX License Expression that is either "L" or equivalent to "L OR E" for some
+acceptable license L from the following list and another SPDX License Expression E.
+
+* BSD-2-Clause: [BSD 2-Clause License](https://opensource.org/licenses/BSD-2-Clause)
+* BSD-3-Clause: [BSD 3-Clause License](https://opensource.org/licenses/BSD-3-Clause)
 * CC0-1.0: [Creative Commons CC0 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/)
-* GNU-All-Permissive: [GNU All-Permissive License](http://www.gnu.org/prep/maintain/html_node/License-Notices-for-Other-Files.html)
+* FSFAP: [FSF All Permissive License](https://www.gnu.org/prep/maintain/html_node/License-Notices-for-Other-Files.html)
```

Comment:
In commit e5748c99971982634, maybe update the link as well, to https://spdx.org/licenses/FSFAP.html